src/bioutils/seqfetcher.py

```diff
@@ -97,7 +98,7 @@ def fetch_seq(ac, start_i=None, end_i=None):
 # Internal functions
 
 
-def _fetch_seq_ensembl(ac, start_i=None, end_i=None):
+def _fetch_seq_ensembl(ac, start_i=None, end_i=None, type=None):
```

Comment:
```suggestion
def _fetch_seq_ensembl(ac, start_i=None, end_i=None, seq_type=None):
```

docstring says `seq_type`, which IMO is a bit better to avoid overriding a builtin

Reply:
Agreed. Done.